manifests/params.pp

```diff
@@ -4,7 +4,9 @@
     'Debian'  => '/etc/ssl',
     'RedHat'  => '/etc/pki',
     'FreeBSD' => '/etc/ssl',
-    default   => fail("ssl module doesn't support OS family ${facts['os']['family']}"),
+    # This will not work on various platforms, e.g. Windows, but it's important
+    # that this not fail so that ssl::cert works on those platforms.
+    default   => '/etc/ssl',
   }
```

Comment:
With this change I would prefer to either see you strip out the lines that match the default.

Reply:
I wanted to be explicit about what was correct. I suppose I could change this to "unsupported" or something.

Reply:
This is usefully explanatory, I think it's the kind of thing that is worth a few extra technically redundant lines for what it adds in readability and ease of understanding.